hazelcast/src/main/java/com/hazelcast/monitor/impl/NearCacheStatsImpl.java

```diff
@@ -41,6 +41,16 @@
             newUpdater(NearCacheStatsImpl.class, "evictions");
     private static final AtomicLongFieldUpdater<NearCacheStatsImpl> EXPIRATIONS =
             newUpdater(NearCacheStatsImpl.class, "expirations");
+    private static final AtomicLongFieldUpdater<NearCacheStatsImpl> PERSISTENCE_COUNT =
```

Comment:
Hm, I could be wrong but aren't the stats updated by only one thread at a time? I guess this is the periodic store run by `DefaultNearCache.StorageTask#run` which already guards against simultaneous runs.
Also, since the fields are not volatile anymore and the reads are not guarded by the same lock, it means that the reads from an another thread could return stale values.